### PR TITLE
charm-gum@0.14.1: Fix `extract_dir`

### LINKED ISSUE #5781: [Bug]: charm-gum@0.14.0 needs `extract_dir`
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

charm-gum

### Expected/Current Behaviour

charm-gum@0.14.0 cannot be installed. It used to be.

### Steps to Reproduce

```console
PS> scoop install charm-gum
…
Extracting gum_0.14.0_Windows_x86_64.zip ... done.
Linking ~\scoop\apps\charm-gum\current => ~\scoop\apps\charm-gum\0.14.0
Creating shim for 'gum'.
Line |
 783 |              $bin = (Get-Command $target).Source
     |                      ~~~~~~~~~~~~~~~~~~~
     | The term 'gum.exe' is not recognized as a name of a cmdlet, function, script file, or executable program. Check the
     | spelling of the name, or if a path was included, verify that the path is correct and try again.
Can't shim 'gum.exe': File doesn't exist.
```

### Possible Solution

#5780

### Scoop and Buckets Version

```console
N/A
```

### Scoop Config

```console
N/A
```

### PowerShell Version

```console
N/A
```

### Additional Softwares

_No response_

### PR DESCRIPTION
**Pull request by YDX-2147483647**
Newer version (updated in https://github.com/ScoopInstaller/Main/commit/340b2b61bfff7942c1038005d036af037830c89f today) of charm-gum has a directory in its archive.

Fixes #5781

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by littleli — APPROVED** [review]

File: bucket/charm-gum.json [hunk 1]
```diff
@@ -19,13 +21,15 @@
         "architecture": {
             "32bit": {
                 "url": "https://github.com/charmbracelet/gum/releases/download/v$version/gum_$version_Windows_i386.zip",
+                "extract_dir": "gum_0.14.0_Windows_i386",
```
Comment (amreus) [inline bucket/charm-gum.json:24]: Use `"extract_dir": "gum_$version_Windows_i386"` here.

File: bucket/charm-gum.json [hunk 2]
```diff
@@ -19,13 +21,15 @@
...
                 "url": "https://github.com/charmbracelet/gum/releases/download/v$version/gum_$version_Windows_i386.zip",
+                "extract_dir": "gum_0.14.0_Windows_i386",
                 "hash": {
                     "url": "$baseurl/checksums.txt",
                     "regex": "($sha256)\\s+gum_[\\d.]+_Windows_i386\\.zip"
                 }
             },
             "64bit": {
                 "url": "https://github.com/charmbracelet/gum/releases/download/v$version/gum_$version_Windows_x86_64.zip",
+                "extract_dir": "gum_0.14.0_Windows_x86_64",
```
Comment (amreus) [inline bucket/charm-gum.json:32]: And `"extract_dir": "gum_$version_Windows_x86_64"` here.

Comment (YDX-2147483647) [reply bucket/charm-gum.json]: Thanks!

**Review by amreus — APPROVED** [review]
👍

**Comment by olivier-hubaut** [comment]
Hello!

Any idea as to when this PR will be merged? 
It has been 3 weeks that gum has been broken due to this. 

As a fallback, we have to pin down the version, which is less than ideal for a workaround.

**Comment by YDX-2147483647** [comment]
> As a fallback

Hi @olivier-hubaut ! May be you can switch to Charm's dedicated bucket for a while, if it's worth it.

```powershell
> scoop uninstall charm-gum
Uninstalling 'charm-gum' (0.13.0).
…

> scoop bucket add charm https://github.com/charmbracelet/scoop-bucket/
…

> scoop search charm-gum
Name      Version Source Binaries
----      ------- ------ --------
charm-gum 0.14.0  main
charm-gum 0.14.0  charm

> scoop install charm/charm-gum

> gum --version
gum version v0.14.0 (295a671)
```

**Comment by YDX-2147483647** [comment]
The new v0.14.1 published, and I've just updated hashes.

**Pull request merged**